hack/update-bindata.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+if [ -z ${KUBE_ROOT} ]; then
+	echo "KUBE_ROOT not detected, setting default."
```

Comment:
I guess what I've been poorly trying to explain is that `KUBE_ROOT` is never set before calling this script, so the conditional and this message aren't very helpful. Look at the Jenkins build logs:
```
Go version: go version go1.6.2 linux/amd64
KUBE_ROOT not detected, setting default.
/go/src/k8s.io/kubernetes/test/e2e/framework
Generated bindata file : 11764 ../../..//test/e2e/generated/bindata.go lines of lovely automated artifacts
```